### ISSUE TITLE
Kernel Matern52 inconsistent with sklearn

### ISSUE THREAD

**Issue by RodrigoAVargasHdz** [dfm/tinygp#31]
Hi,

I've been trying to optimize a GP with the Matern52 kernel. 
During the optimization for a toy 6D problem, I found that the kernel parameters are set to zero (approx 1E-46), so I inspected the kernel and compare its value with `sklearn`. 
To verify that my code works properly I also used the `RBF` kernel but did not fin the same problem. 

- Code:

```
class Matern52_new(kernels.Kernel):
    def __init__(self, scale):
        self.scale = jnp.atleast_1d(scale)

    def evaluate(self, X1, X2):
        X1 = X1 / self.scale
        X2 = X2 / self.scale
        dists = jnp.linalg.norm(X1-X2)
        K = dists * jnp.sqrt(5)
        K = (1.0 + K + K ** 2 / 3.0) * jnp.exp(-K)
        return K
```

```

from sklearn.gaussian_process.kernels import Matern
import numpy as onp
kernel_sk = 1.0 * Matern(length_scale=onp.ones(2), nu=2.5)

kernel = Matern52_ravh(jnp.ones(2))
print(kernel.evaluate(jnp.ones(2),2*jnp.ones(2)))
kernel = kernels.Matern52(jnp.ones(2))
print(kernel.evaluate(jnp.ones(2),2*jnp.ones(2)))
print(kernel_sk(onp.ones((1,2)),2*onp.ones((1,2))))
```

and the results are 
```
(new kernel) 0.3172833639540438
(tinygp) 0.13866021913850426
(sklearn) [[0.31728336]]
```

(extra)
While this custom kernel seems to work, when I plug it into the `tinygp` framework, I still get a `NAN` error when computing the grad of the `neg_log_likelihood`.
You can find my code in the section `## Coustom Matern kernel`
https://colab.research.google.com/drive/151TL_19PrqZIyo7tDmVOuu-j7zFb1hVd?usp=sharing 

Cheers,
Rodrigo

**Comment by dfm** [comment]
Yes - this is the expected behavior as described [here](https://tinygp.readthedocs.io/en/latest/tutorials/multivariate.html) and [here](https://tinygp.readthedocs.io/en/latest/api.html#tinygp.kernels.Matern52) in the docs, and [here](https://github.com/dfm/tinygp/blob/cdd6cfb1baedab52046cf1f6da2384a8e5b245df/src/tinygp/kernels.py#L303-L306) in the code: the Matern kernels are defined with a 1-norm, not a 2-norm. This is different from george, and presumably sklearn (whose interface was based on george), but a sensible default. You're welcome to use your custom kernel, or another reasonable choice would be:

```python
class Matern52(kernels.Kernel):
    def __init__(self, scale):
        self.scale = scale

    def evaluate(self, X1, X2):
        r = jnp.abs((X1 - X2) / self.scale)
        arg = jnp.sqrt(5.0) * r
        return jnp.prod((1.0 + arg + jnp.square(arg) / 3.0) * jnp.exp(-arg))
```
